app/helpers/transport_requisitions_helper.rb

```diff
@@ -0,0 +1,35 @@
+module TransportRequisitionsHelper
+
+	def generate_tr (transport_requisition_params)
```

Comment:
@nathnael I would say this method belongs to the model rather than in a helper class. I think helpers are aimed to be used for logic for the view layer. Not sure if this method qualify for that - it's doing a whole lot of business login in there.